### PR TITLE
Update index.html added my Card

### PR DESCRIPTION
**Pull request by sj5027052**
First time contribution

### CODE REVIEW

**Review by Preet-Sojitra — APPROVED** [review]
LGTM 👍 
Your card will be merged soon.

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+          <!-- ________ Saloni's card START ________  -->
+        <div class="card">
+          <p class="name">Saloni Malaiya</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@Saloni18312589</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://www.twitter.com/Saloni18312589" target="_blank">@Saloni18312589</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindle do the following changes

File: index.html (hunk 1 above)
Comment (sj5027052) [reply index.html]: Done

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@sj5027052, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**